code/modules/shuttles/shuttle.dm

```diff
@@ -328,12 +331,21 @@
 		var/turf/TAS = GetAbove(TS)
 		if(!istype(TAD))
 			continue
+
+		// Check for multi-z shuttles. Don't create a ceiling where the shuttle is about to be.
+		if((istype(TAS) && (get_area(TAS) in shuttle_area)))
+			continue
+
+
 		if(force || (istype(TAD, get_base_turf_by_area(TAD)) || TAD.is_open()))
-			// Check for multi-z shuttles. Don't create a ceiling where the shuttle is about to be.
-			if((istype(TAS) && (get_area(TAS) in shuttle_area)))
-				continue
 			TAD.ChangeTurf(ceiling_type, TRUE, TRUE, TRUE)
 
+		// TODO: Ideally the second check here would remain is_open() rather than a direct type check, but unfortunately we can't do that with only the path.
```

Comment:
it excludes space, which is probably a pretty big deal. the base-turf-by-area check miiiight be handling it in some cases? but like. definitely need to account for both `/turf/space` and `/turf/open`.

Reply:
Addressed